app-frontend/src/app/pages/home/home.html

```diff
@@ -11,7 +11,7 @@ <h1 class="h3">Home</h1>
           </p>
           <a class="btn btn-primary" ui-sref="projects.list">View Projects</a>
           <button class="btn btn-default" ng-click="$ctrl.openCreateProjectModal()">Create a new Project</button>
-          <p><a>Getting started with Projects</a></p>
+          <p><a ng-href="https://help.rasterfoundry.com/creating-projects" target="_blank">Getting started with Projects</a></p>
```

Comment:
It's not necessary to use `ng-href` unless we're doing some binding within the value. So something like this which is just a string can use `href` while something more complex like `{{$ctrl.helpDomain}}/creating-projects` would use `ng-href`.

I don't think you need to change anything you did here, but it's something to be aware of.